src/platform/apollolake/include/platform/memory.h

```diff
@@ -283,7 +275,7 @@
 
 #define SOF_FW_START		(HP_SRAM_VECBASE_RESET + 0x400)
 #define SOF_FW_BASE		(SOF_FW_START)
-#define SOF_FW_SIZE		(0x41000 - 0x400)
+#define SOF_FW_SIZE		(0x90000 - 0x400)
```

Comment:
Can we have a macro for the 0x90000 ? I'm assuming it's maximum possible size for text ?? It would also be good to have macro for 0x400 offset too if you have time.

Reply:
It is a max size for all var-size sections (text/rodata/bss). All "fixed" size sections (stacks, heaps) are moved to the beginning of the mem space (right after mem windows and other fixed size elements) and followed by one large sof_fw_size area defined for linker so it packs everything var-size like .text, .rodata, .data, .bss into that one (with the alignment you noticed that is preserved for the fw manifest representation).

Reply:
@mmaka1 ok, makes sense, but we need to call it something like SOF_FW_MAX_SIZE with a comment e.g.
```
/* max size for all var-size sections (text/rodata/bss) */
#define SOF_FW_MAX_SIZE 0x90000
```

Reply:
@lgirdwood yes, in the next update I will introduce new section for fixed-size things (heaps, stack) and this sof_fw will be for var-size sections. I will include proper comments like in your example.